.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Why ignore this files?

Reply:
No reason. I can remove ".idea" from gitignore if you want.